nightfall-client/src/app.mjs

```diff
@@ -25,7 +25,6 @@ app.use(cors());
 app.use(bodyParser.json({ limit: '2mb' }));
 app.use(bodyParser.urlencoded({ limit: '2mb', extended: true }));
 
-app.get('/healthcheck', (req, res) => res.sendStatus(200));
```

Comment:
Why remove this? We use it to check the server is up.  It will break the ping pong test if you remove it. I can easily fix that but it seems a helpful endpoint.

Reply:
I made this change as I was trying to make the github actions wait for a reaallly "healthy" service (as in, "after the client is synced"). Then i found out about `docker wait` which is a better solution. 

So... I didn't remove it, just put it [here](https://github.com/EYBlockchain/nightfall_3/blob/6ca1b30bf4d5a6dcea379cacafbd663b7a37e281/nightfall-client/src/index.mjs#L20). So the healthcheck only returns 200 after the client is synced. I thought it better reflects the "healthiness" of the service, so I left it there.

Reply:
My bad - hadn't spotted that.  This is a good improvement.